types/b64-lite/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for b64-lite 1.3
+// Project: https://github.com/kevlened/b64-lite#readme
+// Definitions by: JasonHK <https://github.com/JasonHK>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace b64Lite;
```

Comment:
I believe the namespace is incorrect as the module doesn't declare any global variables. Please remove it.

Reply:
It did provide a UMD build, `dist/b64-lite.umd.js`, that exposed `b64Lite` to the global object.

Reply:
Great, in that case merging as is. Thanks for clarifying.